lib/import.js

```diff
@@ -8,7 +8,23 @@ module.exports = (self) => {
         reporting.setTotal(pieces.length);
       }
 
-      const [ piecesToImport, convertErr ] = await self.convertPieces(req, pieces);
+      const {
+        updateKey, updateField, updateKeyErr
+      } = self
+        .checkIfUpdateKey(pieces[0], file.path);
+
+      if (updateKeyErr) {
+        return await self.stopProcess(req, {
```

Comment:
All method names provided by the export module should be prefixed with `export` to prevent conflict with the piece-type module and project level subclasses.

Reply:
all methods have been renamed with `import`